### PR TITLE
Packaging v2

### PR DESCRIPTION
**Pull request by Thovi98**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Thovi98** [comment]
!testme

**Comment by Thovi98** [comment]
@navanchauhan ready for review :)

**Comment by navanchauhan** [comment]
@ericgaspar do we no longer have support for the auto-updater?

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    [install.init_main_permission]
+    type = "group"
+    default = "visitors"
+
+[resources]
+    [resources.sources]
+        [resources.sources.main]
+        url = "https://github.com/advplyr/audiobookshelf/archive/refs/tags/v2.2.17.tar.gz"
+        sha256 = "d873015fbf4b9013002ce247d216fbc1edb22764064ac7c960d8cfa095ddf0cd"
+        autoupdate.strategy = "latest_github_tag"
```
Comment (Thovi98) [inline manifest.toml:43]: @navanchauhan now the autoupdater is integrated in v2 and has to be set in the manifest :)

Comment (navanchauhan) [reply manifest.toml:43]: Ah thank you for pointing that out! LGTM

**Pull request merged**